modules/5_install/install.tf

```diff
@@ -114,6 +120,7 @@ locals {
     fips_compliant           = var.fips_compliant
     rhcos_pre_kernel_options = var.rhcos_pre_kernel_options
     rhcos_kernel_options     = var.rhcos_kernel_options
+    node_labels              = var.node_labels == "" ? local.node_labels : merge(local.node_labels, var.node_labels)
```

Comment:
Did you test this properly?
1. You are comparing a map with a string that will always return `false`.
2. Merge function will always return the same map if other param is `{}` or `null`. No need for the check.

Reply:
@yussufsh Thank you. Updated the changes.